.changeset/tidy-bugs-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': minor
+---
+
+`vite (vocab)`: Adding language chunks to build output
```

Comment:
Does this PR implement full vocab support? If so, I'd call that out in the changeset. Consumers likely have no idea what language chunks are, and nor should they. Should mention the user-facing feature that in the changeset.
```suggestion
`vite`: Support multi-language applications via Vocab
```